apps/total-typescript/src/pages/products/index.tsx

```diff
@@ -172,7 +172,10 @@ const ProductTeaser: React.FC<ProductTeaserProps> = ({
     }
   }, [isPPPAvailable, pppCoupon, isPPPEnabled, setMerchantCoupon])
 
-  if (product.state === 'unavailable' && !purchased) {
+  if (
+    (product.state === 'unavailable' || product.state === 'draft') &&
+    !purchased
+  ) {
```

Comment:
What if we inverted this and only rendered if the `product.state === 'active'`? Then you don't have to enumerate all the different non-active states. And we don't have to worry about something slipping through later when we add a new state at some point in the future.

Reply:
rigth, that's much better. I updated the logic now: https://github.com/skillrecordings/products/pull/1495/commits/bd774bed5f50f3625ef0edc747206a94bbefb16b